sub-modules/Errors.js

```diff
@@ -0,0 +1,17 @@
+class Errors {
```

Comment:
it wouldn't work correctly. if you want to use it at all sub-modules inside, it's better to put 3 copies inside of those package src files